services/py-api/src/utils.py

```diff
@@ -43,3 +45,37 @@ async def stop(self) -> None:
             raise RuntimeError("The AsyncClient has not been initialized!")
 
         await self._async_client.aclose()
+
+class JwtUtility:
+    class UserData(TypedDict):
+        id: int
+        name: str
+        email: str
+        is_admin: bool
+        email_verified: bool
+        team_id: int
+        created_at: str
+        updated_at: str
+         
+
+    @staticmethod
+    def encode(data: UserData) -> str:
+            return encode(data,key=environ["SECRET_KEY"])
+    
+    @staticmethod
+    def decode(token: str) -> Dict[str, Any]:
+     try:
+        decoded_payload=jwt.decode(token, SECRET_KEY, algorithms=["HS256"])
+        return decoded_payload
+     except ExpiredSignatureError:
+            print("The token has expired.")
```

Comment:
You can use the structlog to log this message. Similar to what is done in : 
```Python
except ConnectionFailure as cf:
            LOG.exception("Pinging db failed due to err {}".format(cf))
            return Err("Database is unavailable!")
```

This time you log the message as a warning instead of exception and something like: 

```Python
LOG.warning("Failed to decode the jwt token due to blah blah")
return Err("The jwt token has expired.")
```

Do the same thing for all the following exceptions.

Reply:
Fixed in be223527926be992bf6729e80ee043728b8fa1c5